_posts/2021-01-21-cockpit-236.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Cockpit 236
+author: pitti
+date: 2021-01-07
+tags: cockpit linux
+slug: cockpit-236
+category: release
+summary: Cockpit with SSH key improvements and motd editing
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 236.
+
+In the last two weeks the Cockpit authors primarily worked on some bug fixes, internal refactoring, dropping some obsolete dependencies like [jquery-flot](https://www.npmjs.com/package/jquery-flot), replacing some custom React components with standard [PatternFly](https://www.patternfly.org/v4/) ones, build system simplifications, and test stabilization. These don't have an user-visible effect, but will make Cockpit easier to develop in the future.
```

Comment:
```suggestion
In the last two weeks the Cockpit authors primarily worked on some bug fixes, internal refactoring, dropping some obsolete dependencies like [jquery-flot](https://www.npmjs.com/package/jquery-flot), replacing some custom React components with standard [PatternFly](https://www.patternfly.org/v4/) ones, build system simplifications, and test stabilization. These don't have a user-visible effect, but will make Cockpit easier to develop in the future.
```

....because english


Also: one might argue that bug-fixes are user-visible.  At least, they do more than "make Cockpit easier to develop in the future".

Maybe it makes sense to add a header for this item?  Or at least add a bullet point when you create the tag.

Reply:
Good point -- I'll re-drop the bug fixes. We have never mentioned them in release notes, and we always have some.

Reply:
Moved to separate header (@garrett suggested the same) and fixed the typo, thanks!